Jetsnack/gradle/libs.versions.toml

```diff
@@ -12,7 +12,7 @@ androidx-compose-bom = "2025.05.00"
 androidx-constraintlayout = "1.1.1"
 androidx-core-splashscreen = "1.0.1"
 androidx-corektx = "1.16.0"
-androidx-glance = "1.1.1"
```

Comment:
You need up update the central toml file in `scripts` or else this will be overriden by the next sample update